ppx_mysql.opam

```diff
@@ -11,6 +11,7 @@ build-test: [["dune" "runtest" "-p" name]]
 depends: [
   "alcotest" {test & >= "0.8" & < "0.9"}
   "dune" {build}
+  "ocamlformat" {test}
```

Comment:
You should probably add constraints on the version. Just to be sure.

Reply:
I added a version constraint on OCamlformat 0.7.